tests/system/tests/bash/enable_plugin_coap.bash

```diff
@@ -48,10 +48,21 @@ else
 fi
 
 #
-# Enables the plugin if needed
+# Installs python libraries required by the plugin
 #
+sudo pip3 install -Ir  ${TMP_DIR}/${PLUGIN_COAP_NAME}/python/requirements-coap.txt --no-cache-dir
```

Comment:
Does that mean system test will require to be executed with sudo now?

Reply:
sudo is required, installing without it generates the error : 

```error: [Errno 13] Permission denied: '/usr/local/lib/python3.5/dist-packages/aiocoap-0.3-py3.5.egg-info'

Reply:
we don't want to run ```sudo ./foglamp_test``` 
Please use --user in the above pip installation and remove sudo from here. This is the same difference as make vs sudo make install